rewrite-java-core/src/main/resources/META-INF/rewrite/rewrite.yml

```diff
@@ -6,4 +6,12 @@ recipeList:
   - org.openrewrite.java.ChangePackage:
       oldPackageName: com.azure
       newPackageName: io.clientcore
-      recursive: true
\ No newline at end of file
+      recursive: true
+  - org.openrewrite.text.FindAndReplace:
+      find: 'requires com.azure.core'
+      replace: 'requires com.azure.core.v2'
+      filePattern: '**/module-info.java'
+  - org.openrewrite.text.FindAndReplace:
+      find: 'requires transitive com.azure.core'
+      replace: 'requires transitive com.azure.core.v2'
+      filePattern: '**/module-info.java'
```

Comment:
@srnagar Do we keep all the recipes in on yaml or better to break out then into their respective concepts?

Reply:
It's fine to keep them all in one yaml for now. Some of the more advanced recipes might require their own yaml but simple recipes like this can stay in one yaml.

Reply:
We should add good documentation for each recipe to describe when this will used and what it does.

Reply:
@samvaity Is it worth tracking the changes by checking in the `rewrite.patch` file too?

Reply:
I think thats a good idea. 
Also, worth to adding unit tests following https://docs.openrewrite.org/authoring-recipes/recipe-testing#example-tests